### PR TITLE
cli: Hide access token from the command description

### LINKED ISSUE #11: cli: hide access token from the command description
Right now when one checks a command description, access token is being displayed. It happens because Cobra library displays the default value used for command. We should hide this sensitive information, but still set the value from environmental variable by default.

Example:
```
$ go run . ping --help

Check connection to REANA server.
...
Flags:
  -t, --access-token string   Access token of the current user. (default "XXX")
```

### PR DESCRIPTION
**Pull request by BrunoRosendo**
closes #11

Hides the access token from the command description when using `--help`. Did it by having our own logic for a default value, instead of using Cobra's feature

### CODE REVIEW

File: cmd/info.go [hunk 1]
```diff
@@ -40,6 +40,10 @@ func newInfoCmd() *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			jsonOutput, _ := cmd.Flags().GetBool("json")
 			token, _ := cmd.Flags().GetString("access-token")
+			if token == "" {
+				token = os.Getenv("REANA_ACCESS_TOKEN")
+			}
+
```
Comment (audrium) [inline cmd/info.go:46]: Minor: do we need these empty lines after this check? IMO it logically separates a `token` from `serverURL` which are kind of related, WDYT?

**Review by audrium — COMMENTED** [review]
Congrats with your first contribution!

I left one cosmetic comment, but it works nicely!
Could you please rebase your commit on top of latest changes and add yourself
as an author [here](https://github.com/reanahub/reana-client-go/blob/master/AUTHORS.rst)?

Also, can you add an empty line between your commit and closes statement like described in the [development wiki](https://github.com/reanahub/reana/wiki/Developing#structure-and-style-of-commits)? P.S. usually we keep the commit messages in lowercase, it would be nice to stick to this convention

File: cmd/info.go (hunk 1 above)
Comment (BrunoRosendo) [reply cmd/info.go:46]: Sure, I just thought it was too bloated but maybe it makes sense to put them together, I can just that

**Review by audrium — APPROVED** [review]
Nicely done

**Pull request merged**